### PR TITLE
printfn -> pretty output from Spectre.Console

### LINKED ISSUE #7: Add Colored Output
We can replace all of the `println` with `AnsiConsole.MarkupLine` from https://github.com/spectresystems/spectre.console
as well as all of the `eprintln` with it as well

Also as a bonus, we can add a [Progress](https://spectresystems.github.io/spectre.console/progress) to the up and down commands if we want to go fancy with it

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
Closes #7 (without the bonus).

I moved it to .net 5, but I'm not sure you will like it, so if not, I can revert it to 3.1.

Please, review, @AngelMunoz

### CODE REVIEW

**Comment by AngelMunoz** [comment]
you can leave netcoreapp3.1 as is, I will update to net6 on a different set of commits, I have cut the 6.x.x branch for any maintenance updates we might need before net6 gets rtm

**Comment by WhiteBlackGoose** [comment]
Reverted

File: src/Migrondi/UserInterface.fs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿module UserInterface
```
Comment (AngelMunoz) [inline src/Migrondi/UserInterface.fs:1]: Pretty cool, I'd just go and put everything inline 😅 but this makes it more maintainable, I'm loving it

Comment (WhiteBlackGoose) [reply src/Migrondi/UserInterface.fs:1]: Yep. If you would like to revert back some day, you will replace it in one place.

**Review by AngelMunoz — APPROVED** [review]

**Pull request merged**